sw/host/cryptotest/testvectors/parsers/cryptotest_util.py

```diff
@@ -59,9 +59,9 @@ def str_to_byte_array(s: str) -> list:
     For example, str_to_byte_array("01020a0b") -> [1, 2, 10, 11]
     """
     if s.startswith("0x"):
-        s = s[1:]
+        s = s[2:]
     if len(s) % 2 != 0:
-        raise ValueError(f"String {s} has an odd number of digits; cannot convert into byte array.")
+        s = "0" + s
```

Comment:
This change adds compatibility to make it easier on the test vector parser. If we get an odd-length hex string, we simple append a leading zero character and then parse as usual.